File: specification/appplatform/resource-manager/Microsoft.AppPlatform/stable/2022-04-01/examples-js/Certificates_Delete.js
const { AppPlatformManagementClient } = require("@azure/arm-appplatform");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Delete the certificate resource.
 *
 * @summary Delete the certificate resource.
 * x-ms-original-file: specification/appplatform/resource-manager/Microsoft.AppPlatform/stable/2022-04-01/examples/Certificates_Delete.json
 */
async function certificatesDelete() {
  const subscriptionId = "00000000-0000-0000-0000-000000000000";
  const resourceGroupName = "myResourceGroup";
  const serviceName = "myservice";
  const certificateName = "mycertificate";
  const credential = new DefaultAzureCredential();
  const client = new AppPlatformManagementClient(credential, subscriptionId);
  const result = await client.certificates.beginDeleteAndWait(
    resourceGroupName,
    serviceName,
    certificateName
  );
  console.log(result);
}

certificatesDelete().catch(console.error);

File: specification/appplatform/resource-manager/Microsoft.AppPlatform/stable/2023-12-01/examples-java/Certificates_Delete.java

/**
 * Samples for Certificates Delete.
 */
public final class Main {
    /*
     * x-ms-original-file:
     * specification/appplatform/resource-manager/Microsoft.AppPlatform/stable/2023-12-01/examples/Certificates_Delete.
     * json
     */
    /**
     * Sample code: Certificates_Delete.
     * 
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void certificatesDelete(com.azure.resourcemanager.AzureResourceManager azure) {
        azure.springServices().manager().serviceClient().getCertificates().delete("myResourceGroup", "myservice",
            "mycertificate", com.azure.core.util.Context.NONE);
    }
}
